playpen/mongoengine/dockerimage/rhel-pulp.repo

```diff
@@ -4,6 +4,15 @@
 [pulp-v2-stable]
 name=Pulp 2.8 Stable Builds
 baseurl=http://repos.fedorapeople.org/repos/pulp/pulp/stable/2.8/7Server/$basearch/
+enabled=0
+skip_if_unavailable=0
+gpgcheck=1
+gpgkey=https://repos.fedorapeople.org/repos/pulp/pulp/GPG-RPM-KEY-pulp-2
+
+# Version 2.8 Beta Builds
+[pulp-v2-beta]
+name=Pulp 2.8 Beta Builds
+baseurl=http://repos.fedorapeople.org/repos/pulp/pulp/beta/2.8/7Server/$basearch/
```

Comment:
The docker image does include the script and an installation of pulp code. You can see an ``ADD`` statement for it in the ``Dockerfile``.

To update the script itself, it must be changed in git, and then the docker image just needs to be re-built on docker hub. Docker hub builds the image from our master branch.

To change how the image gets built, such as which packages get installed, requires a similar process. A PR modifies the Dockerfile, or any related files (in this case the repo file), and then the image must be re-built on docker hub.